app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -236,13 +244,28 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
     }
 
     protected open fun startupTelemetryOnCreateCalled(safeIntent: SafeIntent, hasSavedInstanceState: Boolean) {
-        components.appStartupTelemetry.onHomeActivityOnCreate(safeIntent, hasSavedInstanceState)
+        components.appStartupTelemetry.onHomeActivityOnCreate(
+            safeIntent,
+            hasSavedInstanceState, homeActivityInitTimeStampNanoSeconds
```

Comment:
nit: kotlin convention is either have the args all on one line or all the args on different lines

Reply:
fixed